src/detect-fsd-root.ts

```diff
@@ -13,7 +13,7 @@ export async function getLayersCountInFolder(
   const files = await fs.readdir(folderPath);
 
   return files.filter((file) =>
-    layers.some((ignored) => file.endsWith(ignored)),
+    layers.some((ignored) => path.basename(file) === ignored),
```

Comment:
suggestion (non-blocking):
```suggestion
    layers.includes(path.basename(file)),
```